cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestManager.java

```diff
@@ -334,10 +337,35 @@ public void deleteNamespace(Id.Namespace namespace) throws Exception {
   }
 
   @Override
+  public void deleteNamespaceData(NamespaceId namespace) throws Exception {
```

Comment:
I guess only tests that test Namespace APIs. But we should make these available from somewhere. Perhaps not from ``TestBase``, but from a ``NamespaceManager``? What do you think?